test/options-manager.js

```diff
@@ -226,31 +224,32 @@ test('buildConfig: nodeVersion: false', t => {
 	t.is(config.rules['node/prefer-promises/fs'], 'off');
 });
 
-test('buildConfig: nodeVersion: invalid range', t => {
```

Comment:
Removed that test as this case can never actually happens as the validity of the semver range is verified earlier in `mergeOptions`